packages/bsky/tests/auth.test.ts

```diff
@@ -0,0 +1,64 @@
+import AtpAgent from '@atproto/api'
+import { SeedClient, TestNetwork } from '@atproto/dev-env'
+import usersSeed from './seeds/users'
+import { createServiceJwt } from '@atproto/xrpc-server'
+import { Keypair, Secp256k1Keypair } from '@atproto/crypto'
+
+describe('auth', () => {
```

Comment:
these tests _might_ make more sense on `xrpc-server`. thoughts?

Reply:
Yeah, I can see that.  I am inclined to leave this test here since it's an important feature of the appview (and it could easily be implemented incorrectly), but I will add unit tests to xrpc-server for `verifyJwt()` 👍